source/get-started/download-and-install.txt

```diff
@@ -40,12 +40,20 @@ Download and Install
 
    .. step:: Install {+driver-short+}
 
-      Run the following command in your shell to install
-      the driver in your project directory:
+      Run the following commands in your shell to create and activate a virtual
+      environment in which to install the driver:
 
       .. code-block:: bash
 
-         $ python3 -m pip install pymongo=={+patch-version-number+}
+         python -m venv venv
```

Comment:
Strangely enough, one often doesn't have `python` on their machine, and must start with `python3`. Once a virtual env is activated `python` will always be there (typically symbolically linked to python3). So, the first line should be `python3 -m venv venv`. The rest is good, although, as described, one typically uses plain python in the activated environment. The necessary change is only the one python -> python3.

Reply:
Good catch, thanks! Fixed